utility_network/display-content-of-utility-network-container/src/main/java/com/esri/samples/display_content_of_utility_network_container/DisplayContentOfUtilityNetworkContainerController.java

```diff
@@ -0,0 +1,305 @@
+                        graphicsOverlay.getGraphics().add(new Graphic(content.getGeometry(), symbol));
+                      });
+
+                      Geometry firstGraphic = graphicsOverlay.getGraphics().get(0).getGeometry();
+                      double containerViewScale = containerElement.getAssetType().getContainerViewScale();
+
+                      if (graphicsOverlay.getGraphics().size() == 1 && firstGraphic instanceof Point) {
+                        mapView.setViewpointCenterAsync((Point) firstGraphic, containerViewScale).addDoneListener(() -> {
+
+                          // the bounding box, which defines the container view, may be computed using the extent of the features
+                          // it contains or centered around its geometry at the container's view scale
+                          Geometry boundingBox = mapView.getCurrentViewpoint(Viewpoint.Type.BOUNDING_GEOMETRY).getTargetGeometry();
+                          identifyAssociationsWithExtent(boundingBox);
+                          new Alert(Alert.AlertType.INFORMATION, "This feature has no associations").show();
+
+                        });
+
+                      } else {
+                        Geometry boundingBox = GeometryEngine.buffer(graphicsOverlay.getExtent(), 0.05);
+                        identifyAssociationsWithExtent(boundingBox);
+                      }
+
+                    } catch (Exception ex) {
+                      new Alert(Alert.AlertType.ERROR, "Error fetching features for elements.").show();
+                      ex.printStackTrace();
+                    }
+                  });
+
+                } else {
+                  new Alert(Alert.AlertType.ERROR, "No content elements found").show();
+                }
+
+              } catch (Exception ex) {
+                new Alert(Alert.AlertType.ERROR, "Error getting containment associations.").show();
+                ex.printStackTrace();
+              }
+            });
+          }
+
+        } catch (InterruptedException | ExecutionException ex) {
+          new Alert(Alert.AlertType.ERROR, "Error getting result of the query.").show();
+          ex.printStackTrace();
+        }
+      });
+
+    }
+
+  }
+
+  /**
+   * Get associations for the specified geometry and display its associations.
+   *
+   * @param boundingBox the geometry from which to get associations.
+   */
+  private void identifyAssociationsWithExtent(Geometry boundingBox) {
+
+    graphicsOverlay.getGraphics().add(new Graphic(boundingBox, boundingBoxSymbol));
+    mapView.setViewpointGeometryAsync(GeometryEngine.buffer(graphicsOverlay.getExtent(), 0.05));
+
+    // get the associations for this extent to display how content features are attached or connected.
+    ListenableFuture<List<UtilityAssociation>> extentAssociations = utilityNetwork.getAssociationsAsync(graphicsOverlay.getExtent());
```

Comment:
You call `graphicsOverlay.getExtent()` twice in this method so could consider creating a variable to reuse.